test/yagna/helpers/activity.py

```diff
@@ -0,0 +1,73 @@
+"""Activity helpers."""
+
+import os
+from pathlib import Path
+
+from goth.runner import Runner
+
+
+def vm_exe_script(runner: Runner, output_file: str):
+    """VM exe script builder."""
+
+    output_path = Path(runner.web_root_path) / output_file
+    if output_path.exists():
+        os.remove(output_path)
+
+    web_server_addr = f"http://{runner.host_address}:{runner.web_server_port}"
+
+    return [
+        {"deploy": {}},
+        {"start": {}},
+        {
+            "transfer": {
+                "from": f"{web_server_addr}/scene.blend",
+                "to": "container:/golem/resource/scene.blend",
+            }
+        },
+        {
+            "transfer": {
+                "from": f"{web_server_addr}/params.json",
+                "to": "container:/golem/work/params.json",
+            }
+        },
+        {"run": {"entry_point": "/golem/entrypoints/run-blender.sh", "args": []}},
+        {
+            "transfer": {
+                "from": f"container:/golem/output/{output_file}",
+                "to": f"{web_server_addr}/upload/{output_file}",
+            }
+        },
+    ]
+
+
+def wasi_exe_script(runner: Runner, output_file: str):
+    """WASI exe script builder."""
+
+    output_path = Path(runner.web_root_path) / output_file
+    if output_path.exists():
+        os.remove(output_path)
+
+    web_server_addr = f"http://{runner.host_address}:{runner.web_server_port}"
+
+    return [
+        {"deploy": {}},
+        {"start": {"args": []}},
+        {
+            "transfer": {
+                "from": "http://3.249.139.167:8000/LICENSE",
+                "to": "container:/input/file_in",
+            }
```

Comment:
I've probably already forgotten - do we need this transfer step for anything? If so, maybe we should just include this `LICENSE` file in the test's assets so that we don't need to rely on an external server?

Reply:
definitely. Tests shouldn't be related to any central component